flixel/addons/editors/tiled/TiledObject.hx

```diff
@@ -59,11 +59,11 @@ class TiledObject
 	/**
 	 * Whether the object is flipped horizontally.
 	 */
-	public var flippedHorizontally(get, null):Bool;
+	public var flippedHorizontally:Bool;
 	/**
 	 * Whether the object is flipped vertically.
 	 */
-	public var flippedVertically(get, null):Bool;
+	public var flippedVertically:Bool;
```

Comment:
`null`-set access means setting the property is only allowed inside this class. Is there a reason to lift that restriction?

Otherwise, it should be `(default, null)`.